### ISSUE TITLE
Background and foreground colors have insufficient contrast ratio.

### ISSUE THREAD

**Issue by krisu7** [firebase/firebaseui-web#202]
Lighthouse is complaining about contrast of this element
`<span class="firebaseui-idp-text firebaseui-idp-text-long">`
found in `soy/elements.soy`

It gives helpful link [https://developers.google.com/web/tools/lighthouse/audits/contrast-ratio](https://developers.google.com/web/tools/lighthouse/audits/contrast-ratio)

**Comment by AnthonyNahas** [comment]
hi, I implement and angular library for firebase authentication (called [ngx-auth-firebaseui)](https://www.npmjs.com/package/ngx-auth-firebaseui) which has a lot of features... e.g: pick your own material theme incl. primary, accent and warn color. Live demo can be found [here](https://ngx-auth-firebaseui.firebaseapp.com)

**Comment by nicolasgarnier** [comment]
Assigning to @bojeil-google to fix contrast ratio in the default styling.

**Comment by jkevingutierrez** [comment]
Any updates here?

**Comment by dackers86** [comment]
Thank you for taking the time to submit this issue.

A new release is planned shortly, which should resolve this request. 

In the meantime, we will close this issue, if this is not resolved in the new version - please feel free to create a new issue.
